pages/register.tsx

```diff
@@ -275,65 +281,174 @@ export default function Register() {
+                  minutes.
+                  <h2 className="sm:text-2xl text-xl font-semibold mt-5 sm:mb-3 mb-1">
+                    Sponsor Info
+                  </h2>
+                  <div className="flex flex-col">
+                    {sponsorInfoQuestions.map((obj, idx) => (
+                      <DisplayQuestion
+                        key={idx}
+                        obj={obj}
+                        values={values}
+                        onChange={handleChange}
+                      />
+                    ))}
+                  </div>
+                  {/* Resume Upload */}
+                  <div className=" mt-8">
+                    Upload your resume:
+                    <br />
+                    <input
+                      onChange={(e) => handleResumeFileChange(e)}
+                      name="resume"
+                      type="file"
+                      formEncType="multipart/form-data"
+                      accept=".pdf, .doc, .docx, image/png, image/jpeg, .txt, .tex, .rtf"
+                    />
+                    <br />
+                  </div>
+                  {/* Submit */}
+                  <div className="mt-8 text-white">
+                    <button
+                      type="submit"
+                      className="mr-auto cursor-pointer px-4 py-2 rounded-lg bg-primaryDark hover:brightness-90"
+                      onClick={() => setFormValid(!(!isValid || !dirty))}
+                    >
+                      Submit
+                    </button>
+                    {!isValid && !formValid && (
+                      <div className="text-red-600">Error: The form has invalid fields</div>
+                    )}
+                  </div>
+                </section>
+              )}
             </Form>
           )}
         </Formik>
+
+        {/* Pagniation buttons */}
+        <section
+          className={`lg:block flex ${
+            registrationSection == 0
+              ? 'justify-end'
+              : registrationSection >= 4
+              ? 'justify-start'
+              : 'justify-between'
+          } lg:pb-0 pb-8 lg:px-0 sm:px-8 px-6 text-primaryDark font-semibold text-primaryDark font-semibold lg:text-xl md:text-lg`}
+        >
+          {registrationSection > 0 && (
+            <div
+              className="lg:fixed 2xl:bottom-8 2xl:left-8 bottom-6 left-6 inline cursor-pointer select-none"
+              onClick={() => {
+                setRegistrationSection(registrationSection - 1);
+              }}
+            >
+              <ChevronLeftIcon />
+              previous page
+            </div>
+          )}
+
+          {registrationSection < 4 && (
+            <div
+              className="lg:fixed 2xl:bottom-8 2xl:left-8 bottom-6 right-6 inline cursor-pointer select-none"
+              onClick={() => {
+                setRegistrationSection(registrationSection + 1);
+              }}
+            >
+              next page
```

Comment:
when u click next the previous button overlaps
<img width="267" alt="Screenshot 2023-03-02 at 1 30 19 PM" src="https://user-images.githubusercontent.com/47117192/222532839-2caee4e7-70e7-4a29-a057-d4db8f3f7776.png">

Reply:
next page button is on the right side of page in figma

Reply:
what size screen is this in? I'm not getting this issue

Reply:
3072x1920

Reply:
oops I see it. forgot to change some values after copy pasting